FreeRTOS-Plus/Demo/AWS/Device_Shadow_Windows_Simulator/Device_Shadow_Demo/DemoTasks/shadow_demo_helpers.c

```diff
@@ -58,27 +58,29 @@
 /*------------- Demo configurations -------------------------*/
 
 /** Note: The device client certificate and private key credentials are
- * obtained by the transport interface implementation (with Secure Sockets)
- * from the demos/include/aws_clientcredential_keys.h file.
- *
- * The following macros SHOULD be defined for this demo which uses both server
- * and client authentications for TLS session:
- *   - keyCLIENT_CERTIFICATE_PEM for client certificate.
- *   - keyCLIENT_PRIVATE_KEY_PEM for client private key.
+ * obtained by the TLS transport interface implementation from the
+ * demo_config.h file.
  */
 
+#ifndef democonfigCLIENT_PRIVATE_KEY_PEM
+    #error "Please define client private key(democonfigCLIENT_PRIVATE_KEY_PEM) in demo_config.h."
+#endif
+
+#ifndef democonfigCLIENT_CERTIFICATE_PEM
+    #error "Please define client certificate(democonfigCLIENT_CERTIFICATE_PEM) in demo_config.h."
+#endif
+
 #ifndef democonfigMQTT_BROKER_ENDPOINT
-    #define democonfigMQTT_BROKER_ENDPOINT    clientcredentialMQTT_BROKER_ENDPOINT
+    #error "Define the config democonfigMQTT_BROKER_ENDPOINT by following the instructions in file demo_config.h."
```

Comment:
Can you please change to say `Please define democonfigMQTT_BROKER_ENDPOINT in demo_config.h.` so that it is consistent? Thank you.

Reply:
Sure.